.github/workflows/release.yml

```diff
@@ -0,0 +1,147 @@
+      - uses: cachix/cachix-action@v10
+        with:
+          name: crypto-com
+          signingKey: "${{ secrets.CACHIX_SIGNING_KEY }}"
+      - name: build linux arm64 binaries
+        env:
+          PLATFORM: Linux_arm64
+        run: |
+          BUILD_TYPE="tarball"
+          FLAKE="github:${GITHUB_REPOSITORY}/${GITHUB_REF_NAME}"
+
+          nix build -L --extra-platforms aarch64-linux ${FLAKE}#packages.aarch64-linux.chain-maind-$BUILD_TYPE
+          cp result chain-main_${GITHUB_REF_NAME:1}_${PLATFORM}.tar.gz
+
+          sha256sum *.tar.gz > checksums-$PLATFORM.txt
+
+          echo 'FILES<<EOF' >> $GITHUB_ENV
+          ls -1 *.tar.gz >> $GITHUB_ENV
+          ls -1 checksums-*.txt >> $GITHUB_ENV
+          echo 'EOF' >> $GITHUB_ENV
+          cat $GITHUB_ENV
+      - name: upload linux arm64 binaries
+        uses: softprops/action-gh-release@v1
+        with:
+          draft: true
+          files: "${{ env.FILES }}"
+
+  nix-bundle-x86_64-windows:
+    if: ${{ false }}
+    needs: ["nix-bundle-x86_64"]
+    runs-on: windows-latest
+    environment: release    
+    steps:
+      - name: get source
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: setup msys2
+        uses: msys2/setup-msys2@v2
+        with:
+          msystem: MINGW64          
+          update: false
+          install: git msys2-devel base-devel mingw-w64-x86_64-gcc mingw-w64-x86_64-rocksdb  mingw-w64-x86_64-go  mingw-w64-x86_64-jq  mingw-w64-x86_64-dlfcn        
+      - name: get rocksdb & build
+        env:
+          PLATFORM: Windows_x86_64
+        shell: msys2 {0}
+        run: |
+          uname -a
+          export GOROOT=/mingw64/lib/go
+          export GOPATH=$HOME/go
+          export PATH=$GOPATH/bin:$GOROOT/bin:$PATH             
+          export CGO_CFLAGS="-I/mingw64/include/rocksdb" 
+          export CGO_LDFLAGS="-L/mingw64/lib -lrocksdb -lstdc++ -lm -lz -lbz2 -lsnappy -llz4 -lzstd" 
+          echo $PATH
+          go version
+          wget https://mirror.msys2.org/mingw/mingw64/mingw-w64-x86_64-rocksdb-6.26.1-1-any.pkg.tar.zst
+          pacman -U mingw-w64-x86_64-rocksdb-6.26.1-1-any.pkg.tar.zst --noconfirm
+          COSMOS_BUILD_OPTIONS=rocksdb make      
+          ls -la ./build/
+          mv ./build/chain-maind ./build/chain-maind.exe
+          cp /mingw64/bin/libbz2-1.dll ./build 
+          cp /mingw64/bin/libgcc_s_seh-1.dll ./build
+          cp /mingw64/bin/liblz4.dll ./build
+          cp /mingw64/bin/librocksdb.dll ./build
+          cp /mingw64/bin/libsnappy.dll ./build
+          cp /mingw64/bin/libstdc++-6.dll ./build
+          cp /mingw64/bin/libwinpthread-1.dll ./build
+          cp /mingw64/bin/libzstd.dll ./build
+          cp /mingw64/bin/zlib1.dll ./build
+          cd ./build
+          tar cvfz ../chain-main.tar.gz *
+          cd ..
+          ls -la ./chain-main.tar.gz
+          mv ./chain-main.tar.gz chain-main_${GITHUB_REF_NAME:1}-rocksdb_${PLATFORM}.tar.gz
```

Comment:
it doesn't seem to build or fetch the CosmWasm DLL?

Reply:
oh it's skipped: https://github.com/crypto-org-chain/chain-main/pull/793#discussion_r909273757

Reply:
I removed the windows job for now.

Reply:
I updated the issue description to separate it into three steps, let's support arm mac and windows in another PR.